### ISSUE TITLE
Revise menu button examples to declare aria-expanded="false" when collapsed

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#2834]
Update the JavaScript, documentation, and regression tests  for the three menubutton examples to support declaring `aria-expanded=false` when the menus are not visible.

Related to #2808 and #697.

**Comment by Wildebrew** [comment]
Why do menu buttons need an aria-expanded attribute? There is never a state where you can expand/collapse the menu or view the menu while browsing the page. If you activate them menu button you are stuck in the menu until you take action or close it (by pressing escape), so if a button has an aria-haspopup attribute you know that activating it shos a menu and unless you are in that menu it's obviously collapsed.
I find that using aria-expanded on top of aria-haspopup just adds to verbosity/cognitive load without communicating new info.
For an expandable menu item element inside another menu obviously this is another matter altogether, an aria-expanded attribute is vital, my comment only pertains to standalone buttons that, when clicked, trigger the display of a menu widget.

**Comment by mcking65** [comment]
@Wildebrew 

We are making the change based on discussion in #697, which is also explained in #2808.

Esentially, all your points are valid for screen reader users who are using a keyboard but not when using touch.

**Comment by mcking65** [comment]
Closed via #2839. Publication is scheduled for Dec 19, 2023.
